index.html

```diff
@@ -15,13 +15,19 @@
           <nav>
             <img src="images/Enabled.png" alt="" class="hamburger">
           </nav>
+          <ul>
+            <li><a href="#">Portofolio</a></li>
+            <li> <a href="#">About</a> </li>
+            <li><a href="#">Contact</a></li>
+          </ul>
       </section>
     </header>
 
     <main class="container1">
+<!-- INTRODUCTION -->
       <section class="container">
         <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
-        <p class="pr">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="pr">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <h6 class="a">LET’S CONNECT</h6>
         <ul class="social-media-container">
           <li class="i"><a href="#"><img src="images/angellist.png" alt=""></a></li>
```

Comment:
- [ ] OPTIONAL: What if you also add the LinkedIn icon? 😄. You gonna see how much you'll need it next time 😊. I know you can add it. 
![image](https://user-images.githubusercontent.com/80612925/172896091-d9a8993d-268c-4066-97ad-4b02d79baf62.png)   ![image](https://user-images.githubusercontent.com/80612925/172896172-3c0919dc-5faf-4fec-b49d-357f4c52a46b.png)